update.py

```diff
@@ -23,6 +28,27 @@
     'NE': 'n'
 }
 
+def get_overrides():
+    filename = "csv/overrides.csv"
+    print(f"Get overrides from GSheet to {filename}")
+    try:
+        sheet2csv.sheet2csv(id=SHEET_OVERRIDES, range=RANGE_OVERRIDES, api_key=GOOGLE_API_KEY, filename=filename)
+    except Exception as e:
+        print("Failed to import {}".format(filename))
+        raise e
+
+def append_overrides():
+    doctors = pd.read_csv('csv/doctors.csv', index_col=['doctor','type'])
+    overrides = pd.read_csv('csv/overrides.csv', index_col=['doctor','type'])
+
+    print(doctors)
+    print(overrides)
+
+    doctors.join(overrides, rsuffix='_override')
+
+    print(doctors)
+    doctors.to_csv('csv/doctors-overrides.csv')
```

Comment:
This should likely not end up in a different CSV, but in same doctors.csv?

Reply:
Yes, this is just for now so that we can develop new version on stage first with different source CSV.